packages/hub/services/api-router.ts

```diff
@@ -77,6 +81,8 @@ export default class APIRouter {
       parseBody,
       pushNotificationRegistrationsRoute.delete
     );
+    apiSubrouter.get('/notification-preferences/:push_client_id', parseBody, notificationPreferencesRoute.get);
+    apiSubrouter.post('/notification-preferences', parseBody, notificationPreferencesRoute.post);
```

Comment:
The asymmetry here is weird. Perhaps we should have a PUT to /notification-preferences/:push_client_id instead of a POST?

Reply:
Yes, I think that would be better. I added a fix.